internal/controller/standalone_pgadmin/pod_test.go

```diff
@@ -0,0 +1,175 @@
+//go:build envtest
+// +build envtest
```

Comment:
does this need to be an envtest?

Reply:
I don't think so, something left over from the refactor. I'll update that.